docs/triage.md

```diff
@@ -0,0 +1,61 @@
+# Triage role
+
+As we get more issues and pull requests opened on the GitHub CLI, we've decided on a weekly rotation
+triage role. The initial expectation is that the person in the role for the week spends no more than
+1-2 hours a day on this work; we can refine that as needed.
+
+# Incoming issues
+
+just imagine a flowchart
+
+- can this be closed outright?
+  - e.g. spam/junk
+  - close without comment
+- do we not want to do it?
+  - e.g. have already discussed not wanting to do or duplicate issue
+  - comment acknowledging receipt
+  - close
+- do we want to do it? 
+  - e.g. bugs or things we have discussed before
+  - comment acknowledging it
+  - label appropriately
+  - add to project TODO column if appropriate, otherwise just leave it labeled
+- is it intriguing but needs discussion?
+  - label design-needed if amanda is needed, ping
+  - ping engineers if eng needed
+  - ping billy if producty
```

Comment:
swap? ping billy if producty --> ping PM if product

what changed: name to role, and fixed spelling